edb/api/errors.txt

```diff
@@ -138,6 +138,10 @@
 
 0x_FF_00_00_00   ClientError
 0x_FF_01_00_00   ClientConnectionError
+0x_FF_01_01_00   ConnectionFailedError
+0x_FF_01_01_01   ConnectionFailedTemporarilyError
+0x_FF_01_02_00   ConnectionTimeoutError
+0x_FF_01_03_00   AuthenticationFailedError
```

Comment:
How is this `AuthenticationFailedError` different from `AuthenticationError` above?  Also, I think we need to prefix all these with `Client`

Reply:
My mistake. I was looking for it as a subclass of ClientError. Fixed.

Reply:
> Also, I think we need to prefix all these with Client

+1.